components/core/tools/scripts/lib_install/openssl.sh

```
@@ -0,0 +1,43 @@
+cUsage="Usage: ${BASH_SOURCE[0]} <version> [prefix] [openssldir]"
+if [ "$#" -lt 1 ] ; then
+    echo $cUsage
+    exit
+fi
+version="$1"
+prefix="${2:-/opt/openssl-${version}}"
+openssldir="${3:-$prefix}"  # Default to prefix if not specified
+
+package_name=openssh-${version}
+temp_dir=/tmp/${package_name}-installation
+mkdir -p $temp_dir
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)